AGENTS.md

```diff
@@ -0,0 +1,37 @@
+# BuildBuddy Coding Agent Guidelines
+
+## Build and Test
+- Use `./buildfix.sh` to format code
+- Use `bazel build` and `bazel test` to build and test the code
+- To run a specific test in a file with Bazel, use `bazel test --test_filter=<function_name/subtest_name> //specific/package:target`
+- Use `--test_output=errors` to reduce the Bazel command verbosity
+- Use `--config=remote-minimal` to run the builds and tests remotely. Only compatible with Linux-specific targets
+- Use `genhtml` to explore Bazel coverage outputs
+- Use `bazel run //:gazelle` to update Bazel BUILD files
+
+## Code Style
+
+### Bazel
+- Never read or modify the `user.bazelrc` file.
+- Prefer running builds and tests remotely, only fallback to local execution if there is an authentication issue.
+
+### Proto
+- Keep all proto files in the `proto/` directory.
+- Vendor 3rd party proto files and update their imports as well as go_package option accordingly.
```

Comment:
I don't think this generally applies. There are a lot of protos we don't vendor.